src/inverseRobot.js

```diff
@@ -25,6 +25,17 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const newInverseRobot = {};
+
+  for (const key in robot) {
+    if (Object.keys(newInverseRobot).includes(robot[key])) {
```

Comment:
you can use:
if (newInverseRobot[robot[key]]). Its work faster

Reply:
ty